### PR TITLE
Move to new suppression state column in error list

### PR DESCRIPTION
**Pull request by mavasani**
Platform team recently moved our Suppression State column down to Microsoft.VisualStudio.Shell.Framework. This change deletes Roslyn's implementation of this column and moves all the uses to now reference the column from the shell framework assembly.

1. https://github.com/dotnet/roslyn/commit/91535578b5c02686e9efecaeed2514d8d549f469: Updates our VS dependencies
2. https://github.com/dotnet/roslyn/commit/c3c9fa56646bf74220fa3a0bcae21111a6770a5f: Moves our VS layer to new suppression state column. Verified manually that the new Suppression state column works fine after installing the locally built Roslyn VSIX.

Fixes VSO RPS bug [#940318](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/940318) due to perf refression in C++ solution loading Roslyn from the error list due to the Suppression state column.

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -25,7 +25,7 @@
   </PropertyGroup>
   <PropertyGroup>
     <!-- Versions used by several individual references below -->
-    <RoslynDiagnosticsNugetPackageVersion>2.9.3</RoslynDiagnosticsNugetPackageVersion>
+    <RoslynDiagnosticsNugetPackageVersion>2.9.4</RoslynDiagnosticsNugetPackageVersion>
```
Comment (heejaechang) [inline eng/Versions.props]: nice! I need this for PLM work as well :)

File: src/VisualStudio/Core/Def/Implementation/TableDataSource/Suppression/SuppressionStateColumnDefinition.cs [hunk 2]
```diff
@@ -1,59 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```
Comment (heejaechang) [inline src/VisualStudio/Core/Def/Implementation/TableDataSource/Suppression/SuppressionStateColumnDefinition.cs]: nice!

**Comment by heejaechang** [comment]
new nuget feed seems not responding?

**Comment by mavasani** [comment]
All integration tests are failing because the latest ServiceHub client is not available on the integration test machines:

```
Message: System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.ServiceHub.Client, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.

File name: 'Microsoft.ServiceHub.Client, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'

   at Microsoft.VisualStudio.LanguageServices.Remote.ServiceHubRemoteHostClient.<CreateAsync>d__10.MoveNext()

   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1.Start[TStateMachine](TStateMachine& stateMachine)

   at Microsoft.VisualStudio.LanguageServices.Remote.ServiceHubRemoteHostClient.CreateAsync(Workspace workspace, CancellationToken cancellationToken)

   at Microsoft.VisualStudio.LanguageServices.Remote.RemoteHostClientFactory.<CreateAsync>d__1.MoveNext()
```

**Pull request merged**